rendered_notebooks/03_basic_preprocessing_categorical_variables_exercise_02_solution.ipynb

```diff
@@ -78,23 +78,15 @@
    "execution_count": 4,
    "metadata": {},
    "outputs": [
-    {
-     "name": "stderr",
-     "output_type": "stream",
-     "text": [
-      "/home/lemaitre/miniconda3/envs/scikit-learn-tutorial/lib/python3.7/site-packages/sklearn/model_selection/_split.py:1978: FutureWarning: The default value of cv will change from 3 to 5 in version 0.22. Specify it explicitly to silence this warning.\n",
-      "  warnings.warn(CV_WARNING, FutureWarning)\n"
-     ]
-    },
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
       "The different scores obtained are: \n",
-      "[0.87310362 0.87193661 0.87659705]\n",
+      "[0.87398915 0.87194186 0.872543   0.87551188 0.877457  ]\n",
       "The accuracy is: 0.874 +- 0.002\n",
-      "CPU times: user 11.5 s, sys: 249 ms, total: 11.7 s\n",
-      "Wall time: 3.43 s\n"
```

Comment:
Interesting that some things take twice as much time ... maybe worth a closer look